src/model/lineDAO.js

```diff
@@ -0,0 +1,43 @@
+export default class LineDAO {
+  constructor() {
+    this.localStorage = window.localStorage;
+  }
+  getAllLines() {
+    if (!this.localStorage.getItem("lines")) return {};
+    return JSON.parse(this.localStorage.getItem("lines"));
+  }
+  addLine(lineName, lineStartStation, lineEndStation) {
+    let lines = this.getAllLines();
+    if (Object.keys(lines).indexOf(lineName) != -1) return null;
+    lines[lineName] = [lineStartStation, lineEndStation];
+    this.localStorage.setItem("lines", JSON.stringify(lines));
+    return lineName;
+  }
+  deleteLine(lineName) {
+    let lines = this.getAllLines();
+    delete lines[lineName];
+    this.localStorage.setItem("lines", JSON.stringify(lines));
+  }
+  getStationsList(lineName) {
+    let lines = this.getAllLines();
+    let stationsList = lines[lineName];
+    if (stationsList == null) {
+      alert("잘못된 경로입니다.");
+    }
+    return stationsList;
+  }
+  addStationInLine(lineName, stationName, idx) {
+    let lines = this.getAllLines();
+    if (lines[lineName].includes(stationName)) return null;
```

Comment:
```suggestion
    if (lines[lineName].includes(stationName)) { 
      return null;
    }
```
한줄이더라도 {}를 넣어주는게 가독성/오류방지에 좋다고 생각합니다. 코드 품질 관리 프레임워크인 소나큐브(Sonarqube)에서도 기준이 포함돼있고, eslint 기준에 포함돼있기도해서 리뷰했습니다.